content/articles/building-web-application-with-devetreme/index.md

```diff
@@ -1,96 +1,86 @@
+**Note** DevExpress is not free, but to start, you can use the free trial version.
 
 ### Requirements for DevExtreme
 - Node.js v12.6.0 or later
 - npm v6.2.0 or later
 
-### DevExtreme CLI Installation
+### DevExtreme CLI installation
 Run the command in your terminal.
-1. Run `npm i -g devextreme-cli` (Note: This command will install the CLI Globally. Alternatively execute this command [npx](https://www.npmjs.com/) to call the CLI command without global installation)
+1. Run `npm i -g devextreme-cli` (Note: This command will install the CLI Globally. Alternatively execute this command with [npx](https://www.npmjs.com/) to call the CLI command without global installation)
 
-### Creating DevExtreme Application
-The way we create Devextreme application is quite different depending on the javascript framework we are using [React.js, Angular & Vue.js]
+### Creating a DevExtreme application
+The way we create Devextreme applications is quite different depending on the javascript framework we are using (React.js, Angular, or Vue.js).
 
-**Note** to fully understand this lesson you must be familiar with the basic concept and patterns of the framework you want to use. For beginners, I suggest you use the free trial first so you get yourself familiar with DevExteme and its components.
+> To fully understand this lesson you must be familiar with the basic concept and patterns of the framework you want to use. For beginners, I suggest you use the free trial first so you get yourself familiar with DevExteme and its components.
+
+#### For React applications
+ 1. ```npx -p devextreme-cli devextreme new react-app application-name```
+ 2. ```cd application name```
+ 3. ```npm run start```
 
-### For React Application
-```bash
- 1. npx -p devextreme-cli devextreme new react-app application-name
- 2. cd application name
- 3. npm run start
-```
 - [Documentation](https://js.devexpress.com/Documentation/Guide/React_Components/Create_a_DevExtreme_Application/)
 
-### For Angular Application
-```bash
- 1. npx -p devextreme-cli devextreme new angular-app application-name
- 2. cd application name
- 3. npm run start
-```
+#### For Angular applications
+ 1. ```npx -p devextreme-cli devextreme new angular-app application-name```
+ 2. ```cd application name```
+ 3. ```npm run start```
+
 - [Documentation](https://js.devexpress.com/Documentation/Guide/Angular_Components/Getting_Started/Create_a_DevExtreme_Application/)
 
-### For Vue Application
-```bash
- 1. npx -p devextreme-cli devextreme new vue-app application-name
- 2. cd application name
- 3. npm run start
-```
+#### For Vue applications
+ 1. ```npx -p devextreme-cli devextreme new vue-app application-name```
+ 2. ```cd application name```
+ 3. ```npm run start```
 - [Documentation](https://js.devexpress.com/Documentation/Guide/Vue_Components/Create_a_DevExtreme_Application/)
 
 ### For the demonstration purpose in this tutorial I will create a DevExtreme application using my favorite JS framework Angular.
```

Comment:
> ### For the demonstration purpose in this tutorial I will create a DevExtreme application using my favorite JS framework Angular.

This doesn't sound appropriate as a header. I would change it to something else like: *Building an Angular DevExtreme application*, and then add that sentence (*For the demonstration purpose in this tutorial I will create a DevExtreme application using my favorite JS framework Angular.*) as the start of this section.